### PR TITLE
NB View Tab group commands according to LibO

### PR DESCRIPTION
**Pull request by Andreas-Kainz**
First Group app specific commands
Second Group Zoom commands
Third Group visibility toggle

Signed-off-by: andreas kainz <kainz.a@gmail.com>
Change-Id: I822d1ea8ef3460bea9869235b039749dad028c17

image

### CODE REVIEW

**Comment by Andreas-Kainz** [comment]
As there is space at the toggle status bar, would we add there an command to collapse the notebookbar @pedropintosilva ?

**Comment by pedropintosilva** [comment]
> As there is space at the toggle status bar, would we add there an command to collapse the notebookbar @pedropintosilva ?

Good idea! Yes makes sense.

**Comment by pedropintosilva** [comment]
> First Group app specific commands Second Group Zoom commands Third Group visibility toggle
> 
> Signed-off-by: andreas kainz [kainz.a@gmail.com](mailto:kainz.a@gmail.com) Change-Id: I822d1ea8ef3460bea9869235b039749dad028c17
> 
> image

Sounds good but it seems there are two problems with the zoom commands:
1. the reset zoom is using the wrong icon, shouldn't be [this one](https://github.com/CollaboraOnline/online/blob/master/browser/images/lc_zoomreset.svg)
2. The order of those zoom icons. I would prefer to keep the order as in the statusbar: 
<table>
<tr><td rowspan="2">Reset Zoom</td><td colpan="2">Zoom Out</td></tr>
<tr><td >Zoom In</td></tr></table>

**Comment by Andreas-Kainz** [comment]
Zoom reset didn't work as bigtoolitem that's the reason for the updated layout.

Updated icon, yes I also had this in mind. Will be a separate PR cause the icon is the same than before.

**Comment by Andreas-Kainz** [comment]
I could fix all your _very useful_ comments @pedropintosilva 

image

File: browser/css/notebookbar.css [hunk 1]
```diff
@@ -351,7 +351,7 @@
 	margin-inline-end: 5px !important;
 	text-align: center;
 }
-.unotoolbutton.notebookbar.has-label > *{ /*so they stay side by side*/
+#zoomreset, .unotoolbutton.notebookbar.has-label > *{ /*so they stay side by side*/
```
Comment (pedropintosilva) [inline browser/css/notebookbar.css:354]: clever way of doing it : ) without additional JS. Note for myself: Then maybe in the feature we can somehow improve the bigtoolitem or add an additional parameter to apply that while not applying anything else . I need to think about it :thinking:

**Review by pedropintosilva — APPROVED** [review]
Perfect!!

**Pull request merged**